packages/dashboard/src/services/accountsApi.js

```diff
@@ -1,5 +1,5 @@
 import ky from "ky";
 
-const prefix = process.env.NEXT_PUBLIC_SKYNET_DASHBOARD_URL ?? "";
+const prefix = process.env.NEXT_PUBLIC_PORTAL_DOMAIN ? `https://account.${process.env.NEXT_PUBLIC_PORTAL_DOMAIN}` : "";
```

Comment:
I find it odd that we're fine with returning an invalid URI here but since it's not a new thing I won't dig into it.

Reply:
it's not going to be invalid, if it's empty, it's going to use the base uri from browser